src/internal/components/configurationPanels/GeoPushpinConfigurationPanel.tsx

```diff
@@ -13,11 +27,70 @@ export default class GeoPushpinConfigurationPanel extends ConfigurationPanelCont
         });
     }
 
+    protected getControlProperties() {
+        const props = this.props;
+        const groupNearbyPoints = get(props, "properties.controls.groupNearbyPoints", true);
+
+        return {
+            groupNearbyPoints,
+        };
+    }
+
+    protected renderPointsSection() {
+        const { groupNearbyPoints } = this.getControlProperties();
+
+        const { properties, propertiesMeta, pushData, mdObject } = this.props;
+        const controlsDisabled = this.isControlDisabled();
```

Comment:
the name should be consistent: `controls` with/without `s`

Reply:
`const isControlDisabled = this.isControlDisabled();` is fine to me